src/OneExplorer/OneExplorer.ts

```diff
@@ -405,45 +406,42 @@ input_path=${modelName}.${extName}
       const fstat = fs.statSync(fpath);
 
       if (fstat.isDirectory()) {
-        const childNode = new Node(NodeType.directory, [], vscode.Uri.file(fpath));
+        const dirNode = this.createDirectoryNode(fpath);
```

Comment:
(2) Introduce createDirectory/BaseModel/ConfigNode() helpers for readibility

By interoducing some creator functions, enhance readibility of the main code.